tests/http/batching-disabled.yml

```diff
@@ -0,0 +1,82 @@
+---
+name: Batching disabled
+config: !inline
+  upstream:
+    baseURL: "http://jsonplaceholder.typicode.com"
+    httpCache: true
+    batch:
+      delay: 0
+      maxSize: 0
+  server:
+    port: 8000
+    hostname: "0.0.0.0"
+  schema:
+    query: Query
+  types:
+    Query:
+      fields:
+        user:
+          type: User
+          args:
+            id:
+              type: Int
+              required: true
+          http:
+            path: "/users?id={{args.id}}"
+            groupBy: ["id"]
+    User:
+      fields:
+        id:
+          type: Int
+        name:
+          type: String
+        username:
+          type: String
+
+mock:
+  - request:
+      url: http://jsonplaceholder.typicode.com/users?foo=bar&id=1&foo=bar&id=2
+    response:
+      body:
+        - id: 1
+          name: "Leanne Graham"
+        - id: 2
+          name: "Ervin Howell"
+  - request:
+      url: http://jsonplaceholder.typicode.com/users?id=1
+    response:
+      body:
+        - id: 1
+          name: "Leanne Graham"
+  - request:
+      url: http://jsonplaceholder.typicode.com/users?id=2
+    response:
+      body:
+        - id: 2
+          name: "Leanne Graham"
+  - request:
+      url: http://jsonplaceholder.typicode.com/users?id=1&id=2
+    response:
+      body:
+        - id: -1
+          name: "THIS SHOULD NOT BE CALLED"
+  - request:
+      url: http://jsonplaceholder.typicode.com/users?id=2&id=1
+    response:
+      body:
+        - id: -1
+          name: "THIS SHOULD NOT BE CALLED"
```

Comment:
you can drop this as if the mock is not present the test will automatically fail